src/app/operators/dialogs/csv/csv-config/csv-properties/csv-properties.component.ts

```diff
@@ -80,11 +80,11 @@ export class CsvPropertiesComponent implements OnInit, AfterViewInit, OnDestroy
         // {display: 'hours', value: 'h'},
         {display: 'seconds', value: 'seconds'},
     ];
-    intervalTypes: Array<{display: string, value: IntervalFormat}> = [
-        {display: '[Start,+inf)', value: IntervalFormat.StartInf},
-        {display: '[Start, End]', value: IntervalFormat.StartEnd},
-        {display: '[Start, Start+Duration]', value: IntervalFormat.StartDur},
-        {display: '[Start, Start+Constant]', value: IntervalFormat.StartConst},
+    intervalTypes: Array<{display: string, value: IntervalFormat, columns: number}> = [
```

Comment:
Maybe you can extract a type/interface out of this.
And columns describes the number of required columns for specifying this interval type, so maybe you can change the name accordingly.